twitchAPI/chat.py

```diff
@@ -974,6 +975,10 @@ async def _handle_room_state(self, parsed: dict):
         for handler in self._event_handler.get(ChatEvent.ROOM_STATE_CHANGE, []):
             t = asyncio.ensure_future(handler(dat))
             t.add_done_callback(self._task_callback)
+            
+    async def _handle_user_state(self, parsed: dict):
+        self.logger.debug('got user state event')
+        self._mod_status_cache[parsed['command']['channel'][1:]] = 'mod' if parsed['tags']['mod'] == '1' else 'user'
```

Comment:
i removed the broadcaster part because my bot account wont be the one streaming, but will add it back after i figure out how to fix that part.  with just moving it the way you had it i got a NoneType for the `arsed['tags']['badges'].get('broadcaster') is not None`

Reply:
`parsed['tags'].get('badges', {}).get('broadcaster') is not None` should do the trick